discovery_engine_core/core/src/stack/ops/breaking.rs

```diff
@@ -39,6 +39,7 @@ use super::{
 pub(crate) struct BreakingNews {
     client: Arc<Client>,
     markets: Arc<RwLock<Vec<Market>>>,
+    sources: Arc<RwLock<Vec<String>>>,
```

Comment:
:wrench: Favorite sources will only be used in their own stack no need to apply changes here.